dashboard/basicPreferences.source.ts

```diff
@@ -21,50 +26,88 @@ export const basicPreferencesPane: PaneDefinition = {
+@prefix foaf: <http://xmlns.com/foaf/0.1/>.
+@prefix schema: <http:/schema.org/>.
+@prefix ui: <http://www.w3.org/ns/ui#>.
+@prefix vcard: <http://www.w3.org/2006/vcard/ns#>.
+@prefix : <#>.
 
-  :this
-    <http://purl.org/dc/elements/1.1/title> "Basic preferences" ;
-    a ui:Form ;
-    ui:part :powerUser, :developerUser;
-    ui:parts ( :powerUser :developerUser  ).
+solid:User a rdfs:Class;
+  rdfs:label "user"@en, "utilisateur"@fr;
+  rdfs:comment """Any person who might use a Solid-based system""";
+  rdfs:subClassOf foaf:Person, schema:Person, vcard:Individual.
 
-:powerUser a ui:BooleanField; ui:property solid:powerUser;
-  ui:label "Color user input by user".
-:developerUser a ui:BooleanField; ui:property solid:developerUser;
-  ui:label "Expand image URLs inline".
-:newestFirst a ui:BooleanField; ui:property solid:newestFirst;
-  ui:label "Newest messages at the top".
+# Since these options are opt-in, it is a bit strange to have new users opt in
+# That they are new users - also we do not use this class for anything specific
+# yet
+# solid:NewUser a rdfs:Class;
+#  rdfs:label "new user"@en;
+#  rdfs:comment """A person who might use a Solid-based system who has low
+#  level of familarity with technical details.""";
+#  rdfs:subClassOf solid:User.
 
-:inlineImageHeightEms a ui:IntegerField; ui:property solid:inlineImageHeightEms;
-  ui:label "Inline image height (lines)".
+solid:PowerUser a rdfs:Class;
+  rdfs:label "power user"@en;
+  rdfs:comment """A person who might use a Solid-based system
+  who is prepared to be given a more complex interface in order
+  to be provided with more pwerful features.""";
+  rdfs:subClassOf solid:User.
 
+  solid:Developer a rdfs:Class;
+    rdfs:label "Developer";
+    rdfs:comment """Any person who might use a Solid-based system,
+    who has software development skills.""";
+    rdfs:subClassOf solid:User.
 `
-    const preferencesForm = kb.sym('https://solid.github.io/solid-panes/dashboard/basicPreferencesForm.ttl#this')
-    const preferencesFormDoc = preferencesForm.doc()
-    if (!kb.holds(undefined, undefined, undefined, preferencesFormDoc)) { // If not loaded already
-      ($rdf.parse as any)(preferencesFormText, kb, preferencesFormDoc.uri, 'text/turtle', null) // Load form directly
+    function loadData (doc: $rdf.NamedNode, turtle: String) {
+      doc = doc.doc() // remove # from URI if nec
+      if (!kb.holds(undefined, undefined, undefined, doc)) { // If not loaded already
+        ($rdf.parse as any)(turtle, kb, doc.uri, 'text/turtle', null) // Load form directly
+      }
     }
+    const preferencesForm = kb.sym('urn:uuid:93774ba1-d3b6-41f2-85b6-4ae27ffd2597#this')
+    loadData(preferencesForm, preferencesFormText)
+
+    const ontologyExtra = kb.sym('urn:uuid:93774ba1-d3b6-41f2-85b6-4ae27ffd2597-ONT')
+    loadData(ontologyExtra, ontologyData)
+
+    async function doRender () {
+      var context = await UI.authn.logInLoadPreferences({ dom, div: container })
```

Comment:
Why using `var` instead of a more specific syntax such as `const`?

Reply:
Tim uses `var` usually, and I hadn't noticed. I'll review PRs better before submitting them next time - fixed now.